### DISCUSSION TITLE
输出Contents的value为什么不能直接json化呢？

### DISCUSSION THREAD

**Discussion by polaris-gxf** [alibaba/ilogtail#660] [Help] [answered]

**Comment by yyuuttaaoo** [comment]
能否具体说明一下你的问题，输入是什么，使用的是什么插件输出？

**Comment by polaris-gxf** [comment, marked as answer]
非常抱歉，之前没有测试清楚，现在问题已经解决。
建议如下：
          1、 json化文档中没有说清楚必须是，Type:flusher_kafka_v2 结合processors_json
          2、Contents、contents、content 这3个没有清楚示例，开始以为范例写错了，造成了误解。

再次表示感谢！

**Reply by messixukejia** [reply]
如果文档造成了误解，可以提交pr修正下文档。
